File: problems/n-repeated_element_in_size_2n_array/solution.js
const repeatedNTimes = function(nums) {
    let n = nums.length / 2, obj = {}, counts = {}
    for(const num of nums){
        if(!obj[num]) obj[num] = 0
        obj[num]++
        counts[obj[num]] = num
    }
    return counts[n]
}

File: problems/n-repeated_element_in_size_2n_array/solution.py
class Solution:
    def repeatedNTimes(self, nums: List[int]) -> int:
        n, obj = len(nums) / 2, dict()
        for num in nums:
            if num not in obj:
                obj[num] = 0
            obj[num] += 1
            if obj[num] == n:
                return num
